### ISSUE TITLE
Urethrovesical occlusion

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7531]
**Preferred term label:**
Urethrovesical occlusion

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:226730 
PMID: 10484780  1/1 congenital

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
